### DISCUSSION TITLE
roles

### DISCUSSION THREAD

**Discussion by luci53** [supabase/supabase#12692] [Questions]
hii i am new so sorry but how do i get supabase admin role

**Comment by GaryAustin1** [comment]
If you mean the Postgres database role superbase_admin it no longer exists for users.  You now are postgres role (non-superuser) per this security change... https://github.com/supabase/supabase/discussions/9314

**Reply by GaryAustin1** [reply]
It is not clear what you mean by "everytime I do something" or even "supabase admin role".  There is a database role that users used to have access to called `supabase_admin`, is that what your messages is saying?  The link I gave shows a change Supabase made.  You no longer have access to that role, if that is what you are referring to.  

You don't say if this is Supabase hosted or self hosted and don't show the actual error and what you are doing.  

This discussion area is not Supabase support, but users and sometimes Supabase members helping out.

If your instance is caught up in the security change link I provided, and not working, then as the link says you need to contact support.

Otherwise you'll need to provide more info to see if anyone looking at this discussion can help out.

**Comment by luci53** [comment]
Sir everytime I do something it says u need supabase admin role that's why
I contacted you

**Comment by luci53** [comment]
[image: Screenshot 2023-02-26 214604.png]
